src/main/kotlin/com/zepben/evolve/database/sqlite/tables/SqliteTable.kt

```diff
@@ -22,13 +22,6 @@ abstract class SqliteTable {
     @JvmField
     protected var columnIndex = 0
 
-    private var columnSet: SortedSet<Column>? = null
-    private var createTableSql: String? = null
-    private var preparedInsertSql: String? = null
-    private var preparedUpdateSql: String? = null
-    private var createIndexesSql: Collection<String>? = null
-    private var selectSql: String? = null
```

Comment:
AFAIK these were never written to. Were we planning on using them as lazily computed values?